File: BackEnd/Node/models/bikestop.js
const mongoose = require('mongoose');

const BikestopSchema = new mongoose.Schema({
	stationId: {
		type: String,
		unique: true
	},
	stationName: String,
	stationLatitude: Number,
	stationLongitude: Number
});

module.exports = mongoose.model('Bikestop', BikestopSchema);

File: Client/GoHome/app/src/main/java/com/uos/gohome/retrofit2/Bikestop.java
package com.uos.gohome.retrofit2;

import com.skt.Tmap.TMapPoint;

public class Bikestop {
    int rackTotCnt;
    String stationName;
    int parkingBikeTotCnt;
    int shared;
    double stationLatitude;
    double stationLongitude;
    String stationId;
    Object traveltime;

    public TMapPoint getPoint() {
        return new TMapPoint(stationLatitude, stationLongitude);
    }
}
